Meadowlark-js/packages/meadowlark/test/http/remote.deployment.test.http

```diff
@@ -0,0 +1,255 @@
+# -----------------------------------------------------------------------------
+
+
+### Create a School
+POST http://{{hostname}}/dev/v3.3b/ed-fi/schools
+Authorization: bearer {{token}}
+
+{
+    "schoolId": 122,
+    "nameOfInstitution": "School122",
+    "educationOrganizationCategories" : [
+        {
+            "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Other"
+        }
+    ],
+    "schoolCategories": [
+        {
+             "schoolCategoryDescriptor": "uri://ed-fi.org/SchoolCategoryDescriptor#All Levels"
+        }
+    ],
+    "gradeLevels": []
+}
+
+
+### View all schools - should be one with id 7a5cf3f4a68015c0922e24c73401a21e9fd1767ef60c0b3300f2301e
+GET http://{{hostname}}/dev/v3.3b/ed-fi/schools
+Authorization: bearer {{token}}
+
+
+### Query for a school named 'School1' - should not return any
+GET http://{{hostname}}/dev/v3.3b/ed-fi/schools?nameOfInstitution=School1
+Authorization: bearer {{token}}
+
+
+### Query for a school named 'School122' - should return result
+GET http://{{hostname}}/dev/v3.3b/ed-fi/schools?nameOfInstitution=School122
+Authorization: bearer {{token}}
+
+
+### Attempt to update the school with an invalid educationOrganizationCategoryDescriptor - should fail
+### (Cold start for PUT means this may take up to 5 seconds.)
+PUT http://{{hostname}}/dev/v3.3b/ed-fi/schools/7a5cf3f4a68015c0922e24c73401a21e9fd1767ef60c0b3300f2301e
+Authorization: bearer {{token}}
+
+{
+    "schoolId": 122,
+    "nameOfInstitution": "A School",
+    "educationOrganizationCategories" : [
+        {
+            "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#Invalid"
+        }
+    ],
+    "schoolCategories": [
+        {
+             "schoolCategoryDescriptor": "uri://ed-fi.org/SchoolCategoryDescriptor#All Levels"
+        }
+    ],
+    "gradeLevels": []
+}
+
+
+### Attempt to update the school with a valid educationOrganizationCategoryDescriptor - should succeed
+PUT http://{{hostname}}/dev/v3.3b/ed-fi/schools/7a5cf3f4a68015c0922e24c73401a21e9fd1767ef60c0b3300f2301e
+Authorization: bearer {{token}}
+
+{
+    "schoolId": 122,
+    "nameOfInstitution": "A School",
+    "educationOrganizationCategories" : [
+        {
+            "educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"
+        }
+    ],
+    "schoolCategories": [
+        {
+             "schoolCategoryDescriptor": "uri://ed-fi.org/SchoolCategoryDescriptor#All Levels"
+        }
+    ],
+    "gradeLevels": []
+}
+
+
+### Attempt to delete the school - should succeed
+### (Cold start for DELETE means this may take up to 5 seconds.)
+DELETE http://{{hostname}}/dev/v3.3b/ed-fi/schools/7a5cf3f4a68015c0922e24c73401a21e9fd1767ef60c0b3300f2301e
+Authorization: bearer {{token}}
+
+
+### View all schools - should be none
+GET http://{{hostname}}/dev/v3.3b/ed-fi/schools
```

Comment:
I just saw eventual consistency in action:

1. DELETED - saw 204
2. Very quickly hit this next GET - returned a record
3. Then hit the GET again - empty array.

Fascinating! I'm adding a comment about this.